funcx/errors.py

```diff
@@ -29,7 +29,7 @@ def __repr__(self):
 class MalformedResponse(FuncxError):
     """ FuncX remote service responded with a Malformed Response
     """
-    def __init__(self, task_id, response):
+    def __init__(self, response):
         self.response = response
 
```

Comment:
Shouldn't we should store task_id?

Reply:
The `MalformedResponse` isn't using it for anything, and currently the only place `MalformedResponse` is being used is in case the task id isn't in the response from the server.